docs/operators/filter.md

```diff
@@ -0,0 +1,49 @@
+Filter
+=============
+
+## Operator Objective
+This operator receives an POJO ([Plain Old Java Object](https://en.wikipedia.org/wiki/Plain_Old_Java_Object)) as an incoming tuple
+and based on the filter condition it shall emit the filtered POJO on one output port and the rest on another output port.
```

Comment:
Can you reword it as follows:
"...it emits filtered tuples on one output port and rest on another output port"